commit b8715d57c7e301c36e8949938676b56b4238197a
Author: Umaro <umaro@narshe.net>
Date:   Tue Jun 21 19:19:33 2011 +0200

    msm: timer: remove set_next_event late print
    
        This case is properly handled in the generic code, and there is no need
        for us to print this ourselves.
    
        Change-Id: I80cd1fdbd2927f703d8902ff414cb5351f6fff70
        Signed-off-by: Jeff Ohlstein <johlstei@codeaurora.org>

diff --git a/arch/arm/mach-msm/timer.c b/arch/arm/mach-msm/timer.c
index 7d9541d..dafeb0a 100644
--- a/arch/arm/mach-msm/timer.c
+++ b/arch/arm/mach-msm/timer.c
@@ -169,18 +169,10 @@ static int msm_timer_set_next_event(unsigned long cycles,
 	clock->last_set = now;
 	clock->alarm_vtime = alarm + clock->offset;
 	late = now - alarm;
-	if (late >= (int)(-clock->write_delay << clock->shift) && late < DGT_HZ*5) {
-		static int print_limit = 10;
-		if (print_limit > 0) {
-			print_limit--;
-			printk(KERN_NOTICE "msm_timer_set_next_event(%lu) "
-			       "clock %s, alarm already expired, now %x, "
-			       "alarm %x, late %d%s\n",
-			       cycles, clock->clockevent.name, now, alarm, late,
-			       print_limit ? "" : " stop printing");
-		}
+	if (late >= (int)(-clock->write_delay << clock->shift) &&
+	    late < clock->freq*5)
 		return -ETIME;
-	}
+
 	return 0;
 }
 
